src/docs/03-Code Guidelines/02-HTML-Guidelines.md

```diff
@@ -1,2 +1,129 @@
+-  [Spaces and equal signs](#spaces-and-equal-signs)
+-  [Boolean attributes](#boolean-attributes)
+-  [Break spaces](#break-spaces)
+-  [Break element](#break-element)
+
+## HTML5 standards
+
+Always write HTML that follows the standards defined by the W3C and the WHATWG. To test if your code follows these standards, there are multiple tools available, among them [The W3C validation service](https://validator.w3.org/), where you can copy/paste HTML code. Another option is a browser extension such as [The Web Developer Toolbar](https://chrispederick.com/work/web-developer/), that allows you to validate HTML that is on a local machine/server.
+
+## Semantic HTML
+
+Semantic HTML or semantic markup is HTML that introduces meaning to the web page rather than just presentation. For example, a `<p>` tag indicates that the enclosed text is a paragraph where a `<div>` or a `<span>` element indicates nothing.
+
+Semantic HTML provides:
+
+-  More information to search engines (SEO)
+-  More accessible content (Accessibility)
+-  Help for screenreader users to understand the contents structure and gives it meaning (Accessibility)
+-  Meaning to the browser but also for other developers (readability)
+
+## Indentation
+
+Indent by 4 spaces at all time. Don’t use tabs or mix tabs and spaces for indentation.
+
+**Bad example**
+
+      <section>
+        <p>This is a paragraph.</p>
+      </section>
+
+**Good example**
+
+      <section>
+          <p>This is a paragraph.</p>
+      </section>
+
+## Encoding
+
+Make sure your editor uses UTF-8 as character encoding. Specify the encoding in HTML templates and documents like this.
+
+      <meta charset="utf-8" />
+
+## Lower case elements and attributes
+
+Always use lowercase for elements and attributes. It looks cleaner and improves readability.
+
+**Bad example**
+
+      <SECTION>
+          <p ID="test">This is a paragraph.</p>
+      </SECTION>
+
+**Good example**
+
+      <section>
+          <p id="test">This is a paragraph.</p>
+      </section>
+
+## Quote attribute values
+
+HTML attributes must always use double quotes (").
+
+**Bad example**
+
+      <table class=classname>
+      <table class='classname'>
+
+**Good example**
+
+      <table class="classname">
+
+## Image attributes
+
+Always use the alt attribute with images. It is important for accessibility issues.
+Informative elements should aim for short, descriptive alternate text. Decorative elements can be ignored with an empty alt attribute.
+
+**Bad example**
+
+      <img src="sc.png" /> <!-- Screenreaders will read the scr attribute if no alternative text is added. Never do this -->
+
+**Good examples**
+
+      <img src="sc.png" alt="" /> <!-- Screenreaders will skip/ignore this image. Useful if the image is only decorative -->
+      <img src="sc.png" alt="Sitecore" /> <!-- Screenreaders will read the alternative text "Sitecore" -->
+
+## Spaces and equal signs
+
+Spaces around equal signs is Bad example

```

Comment:
I think we should leave this one out as it's not valid HTML anyway, and we've already specified that above.